spec/models/participant_spec.rb

```diff
@@ -20,27 +20,126 @@
       end
     end
   end
-  describe 'total score' do
-    it 'should be zero when initialized' do
+  describe 'total_score' do
+    it 'initializes to zero' do
       expect(participant.total_score).to eql 0
     end
-    it 'should increase by quiz score' do
+    it 'increases by quiz score' do
       quiz_score = create :quiz_score
       expect { participant.update quiz_score: quiz_score }
         .to change { participant.total_score }
         .by(quiz_score.score)
     end
-    it 'should increase by circle check score' do
+    it 'increases by circle check score' do
       circle_check_score = create :circle_check_score
       expect { participant.update circle_check_score: circle_check_score }
         .to change { participant.total_score }
         .by(circle_check_score.score)
     end
-    it 'should increase by onboard judging score' do
+    it 'increases by onboard judging score' do
       onboard_judging = create :onboard_judging
       expect { participant.update onboard_judging: onboard_judging }
         .to change { participant.total_score }
         .by(onboard_judging.score)
     end
   end
+  describe 'maneuver_score' do
+    it 'returns the maneuver score' do
+      expect(participant.maneuver_score).to eql 0
+    end
+  end
+  describe 'display_information' do
+    let!(:participant) { create :participant }
+    context 'displays bus information' do
+      it 'returns valid string' do
+        expected = participant.bus.number
+        expect(participant.display_information(:bus)).to eql expected
+      end
+    end
+    context 'displays participant information' do
+      it 'returns valid string' do
+        expect(participant.display_information(:name)).to eql participant.name
+      end
+    end
+    context 'displays number information' do
+      it 'returns valid string' do
+        expected = '#' + participant.number.to_s
+        expect(participant.display_information(:number)).to eql expected
+      end
+    end
+    context 'displays name' do
+      it 'shows name' do
+        participant.update! name: 'Person'
+        expect(participant.display_information(:name)).to eql 'Person'
+      end
+    end
+    context 'displays three options' do
+      it 'splits up options' do
+        exp = participant.name.to_s + ' (' + participant.bus.number +
+              ', #' + participant.number.to_s + ')'
+        expect(participant.display_information(:name, :bus, :number)).to eql exp
+      end
+    end
+  end
+  describe 'next_number' do
+    it 'returns the last non-nil participant number' do
+      create :participant
+      p_2 = create :participant
+      expect(Participant.next_number).to eql p_2.number + 1
+    end
+  end
 end
+describe 'scoreboard order' do
+  context 'total score' do
+    it 'sorts participants by total score' do
+      per = create :onboard_judging, :perfect
```

Comment:
It's a good way to set up the test - although, given that none of your lines of code come even *close* to 80 characters, I would recommend more longer, more descriptive variable names to make it easier to read.